descriptors/ModuleDescriptor-template.json

```diff
@@ -1057,10 +1057,14 @@
     },
     {
       "id": "oaipmhview",
-      "version": "1.0",
+      "version": "2.0",
```

Comment:
Why is this change compatibility breaking? Is it because the `oai-pmh-view/instances` endpoint has become a POST rather than a GET?

Reply:
Because the old endpoint is working differently in this version of API

Reply:
I fixed version to emphasize that changes made just extend existing particular API related to instances retrievement.

Reply:
@oleksandr-yatsenko1 What do you mean you fixed the version? 

Unless, the existing endpoint is to be maintained, as @mrdpopov suggests, then it is a compatibility breaking change.

Reply:
The existing endpoint does not need to be maintained, au contraire it needs to be changed. It is only used in mod-oai-pmh.

Reply:
@mrdpopov Indeed, and given it is changing in an incompatible way, it is a breaking compatibility change. I only wanted to confirm my understanding. Thank you for doing so.